webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtOpenstack.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import { sprintf, translate as __ } from '../../../../../../common/I18n';
+
+const VirtOpenstack = ({ vm }) => (
+  <>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Availability zone')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.availability_zone}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    {vm.all_addresses.map(address => (
+      <DescriptionListGroup>
+        <DescriptionListTerm>
+          {sprintf(__('%s network'), address.pool)}
+        </DescriptionListTerm>
+        <DescriptionListDescription>
+          {sprintf(__('%(ip)s (floating), %(fixed_ip)s (fixed ip)'), {
```

Comment:
Would it be better to spit this into two separate items? One for floating and one for fixed?